jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -54,7 +54,7 @@ public Builder setLabel(String label) {
      * add {@code myfile} to the container to be accessed at {@code /path/in/container}.
      *
      * @param sourceFile the source file to add to the layer
-     * @param pathInContainer the path to add the source files to in the container file system
+     * @param pathInContainer the path to add the source file to in the container file system
```

Comment:
This can be either a directory or a file, but either would be added to that exact path. Directory contents would not be added.

`hold` the file might sound like it is a directory containing the file. Maybe like `the exact path for the file in the container file system`?